setup.py

```diff
@@ -27,6 +27,7 @@
         "junitparser==1.4.0",
         "jinja_markdown",
         "htmllistparse==0.5.2",
+        # latest version of ibm-cloud-sdk-core(i.e 3.15.1) only supports python>=3.7
```

Comment:
Would it be possible to reword the comment?

```
# 3.15.0 is the last version supporting python 3.6
```

Reply:
done